gravitybee/__init__.py

```diff
@@ -458,14 +458,16 @@ def _create_hook(self):
 
         # 1 - extra data
         hook += "# collection extra data, if any (using --extra-data option)"
-        for data in self.args.extra["data"]:
-            hook += "\ndatas.append(('"
-            hook += self.args.directories["pkg"] + os.sep
-            if self.args.directories["src"] != '.':
-                hook += self.args.directories["src"] + os.sep
-            hook += self.args.info["pkg_name"] + os.sep + data
-            hook += "', '" + self.args.info["pkg_name"] + "/" + data + "'))"
-            hook += "\n\n"
+        # checks if data attribute exists
+        if self.args.extra["data"]:
```

Comment:
Instead of the `if`, can we just use an empty list as a default?

```suggestion
        for data in self.args.extra.get("data", []):
``````

Reply:
@salman2135 oops, looks like the suggestion wasn't generated in a way that could be accepted literally... we now have two loops... 🤦 can you take a look and clean that up so the _only_ change in the diff is using `.get()` to set a default?